.changeset/green-crabs-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": minor
+---
+
+Added container div to `StatusAdornment`
```

Comment:
Why was this change needed?

Reply:
I thought since the original class name is now being applied to the outer div